crates/net/rpc-builder/src/lib.rs

```diff
@@ -182,13 +185,44 @@ impl RpcServerBuilder {
     /// This consumes the builder and returns a server.
     ///
     /// Note: The server ist not started and does nothing unless polled, See also
-    pub async fn build(self) -> Result<RpcServer, RpcError> {
-        todo!()
+    pub async fn build(self) -> Result<Vec<RpcServer>, RpcError> {
+        let mut servers = Vec::new();
+        let socket_addr = self.http_ws_addr.unwrap();
+        let ips_endpoint = self.ipc_server_path.unwrap();
```

Comment:
These should be unwrap or a default value. For http/ws 8545, for ipc should be in the global datadir? https://github.com/paradigmxyz/reth/blob/acac82fcb3bf31c2e70fe1aba0e9eca6c7ad021f/bin/reth/src/dirs.rs#L13-L22

Reply:
Not sure, so I'll leave ti as unwrap() for now.